File: python/pangram/pangram.py
import string
def is_pangram(word):
    alpa = list(string.ascii_lowercase)
    word = list(word.replace(' ','').lower())
    for letter in word:
        try:
            i = alpa.index(letter)
            alpa.remove(str(letter))
        except:
            print(' ')
    if len(alpa) == 0:
        return True
    return False

File: php/pangram/pangram.php
<?php

function isPangram($phrase)
{
  $alpa = range('a', 'z'); // gets the length of alpabeth
  $phrase = str_split($phrase);

  foreach($phrase as $letter)
  {// array_Search is like indexOf
      if(($index = array_search(strtolower($letter), $alpa)) !== false) // if there is an index
      {
        unset($alpa[$index]);
      }
  }


  return sizeof($alpa) == 0 ? true: false;
}
